documentation/03_resources/07-faq.html.md

```diff
@@ -15,7 +15,7 @@ No, HaxeFlixel abstracts it completely.
 
 - [@SeiferTim's](https://twitter.com/SeiferTim) [Dungeon Crawler Tutorial](http://haxeflixel.com/documentation/tutorials/)
 
-- [Kasoki's](https://github.com/kasoki/GameMechanicExplorer-HaxeFlixel) [Game Mechanic Explorer port](http://gme.kasoki.de/)
+- [Kasoki's](https://github.com/atomicptr/GameMechanicExplorer-HaxeFlixel) [Game Mechanic Explorer port](http://gme.qr9.de/)
```

Comment:
Should `Kasoki` be changed to `atomicptr` as well?

Reply:
@Gama11 Good call, I'll go ahead and update that. 👍